lib/MetaCPAN/Script/Release.pm

```diff
@@ -112,15 +111,9 @@ sub run {
                 MetaCPAN::Util::author_dir( $d->cpanid ),
                 $d->filename,
             );
-            my $ua = LWP::UserAgent->new(
-                parse_head => 0,
-                env_proxy  => 1,
-                agent      => 'metacpan',
-                timeout    => 30,
```

Comment:
These options are also not set in the `ua` builder sub, with the exception of `env_proxy`.  I think it would at least be helpful to set the `agent`.  All of these have corresponding setter methods, so we could preserve them if we need to.